bitnami/gitea/1/debian-12/rootfs/opt/bitnami/scripts/gitea-env.sh

```diff
@@ -60,6 +60,8 @@ gitea_env_vars=(
     GITEA_SMTP_FROM
     GITEA_SMTP_USER
     GITEA_SMTP_PASSWORD
+    GITEA_OAUTH2_CLIENT_AUTO_REGISTRATION
```

Comment:
What if we call it `GITEA_OAUTH2_CLIENT_AUTO_REGISTRATION_ENABLED`?

Reply:
Sure!